### ISSUE TITLE
Association erros - BugSnag Error (Avni Client)

### ISSUE THREAD

**Issue by petmongrels** [avniproject/avni-client#1015]
note there are multiple different org users who are facing this issue.

**BugSnag Error**
https://app.bugsnag.com/samanvay-research-and-development-foundation/avni-client/errors/64746c0adbe2bd00087e6ad3

```
Error Fatal error: LocationMapping{uuid='8477f116-9400-4427-b15c-be56100045df'} is unable to find AddressLevel{uuid='df0cc43b-93c5-47f1-a4f2-57514b9c1daf'} 
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/openchs-models/dist/error/SyncError.js:10:27 SyncError
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/openchs-models/dist/BaseEntity.js:110:33 value
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/openchs-models/dist/AddressLevel.js:194:55 value
    /Users/vinay/code/sf/avni-client/packages/openchs-android/src/service/SyncService.js:315:96 
    [native code] map
    /Users/vinay/code/sf/avni-client/packages/openchs-android/src/service/SyncService.js:315:17 _class2
    /Users/vinay/code/sf/avni-client/packages/openchs-android/src/service/SyncService.js:342:66 _class2
    [native code] value
    /Users/vinay/code/sf/avni-client/packages/openchs-android/src/service/rest/ConventionalRestClient.js:86:28 processResponse
    /Users/vinay/code/sf/avni-client/packages/openchs-android/src/service/rest/ConventionalRestClient.js:94:28 
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/react-native/node_modules/promise/setimmediate/core.js:37:14 tryCallOne
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/react-native/node_modules/promise/setimmediate/core.js:123:25 
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/react-native/Libraries/Core/Timers/JSTimers.js:248:17 
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/react-native/Libraries/Core/Timers/JSTimers.js:112:6 _callTimer
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/react-native/Libraries/Core/Timers/JSTimers.js:162:15 _callReactNativeMicrotasksPass
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/react-native/Libraries/Core/Timers/JSTimers.js:413:11 callReactNativeMicrotasks
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:391:11 value
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:133:11 
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:368:8 value
    /Users/vinay/code/sf/avni-client/packages/openchs-android/node_modules/react-native/Libraries/BatchedBridge/MessageQueue.js:132:17 value
    [native code] value
    [native code] value
```

**Management of BugSnag error and BugSnag**
https://avni.readme.io/docs/how-to-use-bugsnag

**Analysis updates**:
We found three kinds of association errors:
1. `LocationMapping{uuid='8477f116-9400-4427-b15c-be56100045df'} is unable to find AddressLevel{uuid='df0cc43b-93c5-47f1-a4f2-57514b9c1daf'} ` - This has happened in AKRSP UAT not in prod org. There were manual changes to list of all locations recently. This could be the root cause of the problem. No fixes are warranted because this is a UAT org.
2. `ProgramEnrolment{uuid='aa46098b-25a1-4563-b334-bd7da2ad67e0'} is unable to find Individual{uuid='95d0c758-2c9a-4131-b638-3ccb3dec6aa3'} ` - We created a user suhas@apfodisha with similar sync settings and user permissions. Sync went through without errors. Unlike tickets, where we contact the user,  the affected user has not reported an issue, this might not warrant more attention until a report is seen. We can concat the user directly if the need is felt.
3.  `Encounter{uuid='0c44f17e-adaf-4bc1-8461-a0f883ea3621'} is unable to find Individual{uuid='026effd0-a2ca-4699-ba7a-39cb90be257d'} ` - Same as 2. User created: maha@ gdgsgom

**Comment by ashusvnath** [comment]
should all association errors need to be looked into or only the mentioned one?

**Comment by petmongrels** [comment]
all the ones which are reported in this bugsnag issue (one per org is sufficient)

**Comment by ashusvnath** [comment]
**Analysis updates**:
We found three kinds of association errors:
1. `LocationMapping{uuid='8477f116-9400-4427-b15c-be56100045df'} is unable to find AddressLevel{uuid='df0cc43b-93c5-47f1-a4f2-57514b9c1daf'} ` - This has happened in AKRSP UAT not in prod org. There were manual changes to list of all locations recently. This could be the root cause of the problem. No fixes are warranted because this is a UAT org.
2. `ProgramEnrolment{uuid='aa46098b-25a1-4563-b334-bd7da2ad67e0'} is unable to find Individual{uuid='95d0c758-2c9a-4131-b638-3ccb3dec6aa3'} ` - We created a user suhas@apfodisha with similar sync settings and user permissions. Sync went through without errors. Unlike tickets, where we contact the user,  the affected user has not reported an issue, this might not warrant more attention until a report is seen. We can concat the user directly if the need is felt.
3.  `Encounter{uuid='0c44f17e-adaf-4bc1-8461-a0f883ea3621'} is unable to find Individual{uuid='026effd0-a2ca-4699-ba7a-39cb90be257d'} ` - Same as 2. User created: maha@ gdgsgom
